packages/delivery-fetch/delivery.js

```diff
@@ -0,0 +1,44 @@
+import payload from '@bugsnag/core/lib/json-payload'
+
+const delivery = (client, fetch = global.fetch) => ({
+  sendEvent: (event, cb = () => {}) => {
+    const url = client._config.endpoints.notify
+
+    fetch(url, {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+        'Bugsnag-Api-Key': client._config.apiKey,
```

Comment:
Should this be checking for an API key on the event first? It's possible that it may have been overridden in an `onError` callback

Reply:
ahh, understood! updated.